flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -116,6 +117,8 @@ protected void setupFlinkConfig() {
         flinkConfig.set(JobManagerOptions.PORT, RPC_PORT);
         flinkConfig.set(BlobServerOptions.PORT, Integer.toString(BLOB_SERVER_PORT));
         flinkConfig.set(KubernetesConfigOptions.JOB_MANAGER_CPU, JOB_MANAGER_CPU);
+        flinkConfig.set(
```

Comment:
I think we could leave the `KubernetesConfigOptions.JOB_MANAGER_CPU` as default since we do not verify the resource limit in the tests of `Fabric8FlinkKubeClientTest`.